packages/desmos-std/src/profiles/mocks.rs

```diff
@@ -0,0 +1,117 @@
+use crate::profiles::{
+    models_app_links::{AppLinkResult, ApplicationLink, CallData, Data, OracleRequest},
+    models_blocks::UserBlock,
+    models_chain_links::{ChainConfig, ChainLink, ChainLinkAddr, Proof, Signature},
+    models_common::PubKey,
+    models_dtag_requests::DtagTransferRequest,
+    models_profile::{Account, Pictures, Profile},
+    models_relationships::Relationship,
+};
+use cosmwasm_std::{Addr, Binary, ContractResult};
+
+pub trait MockQueriesProfiles<T> {
```

Comment:
I think this trait could be moved to the src/query_mocks.rs with name `MockQueries`.